### ISSUE TITLE
Prevent users from creating 'index' pages?

### ISSUE THREAD

**Issue by brylie** [wagtail/wagtail#4841]
Apologies if this has already been answered, but I couldn't find any related issue.

While working through the Wagtail tutorial, we create a `BlogIndexPage`, and subsequently Blog posts. Once the Blog index is created, there is no need to create another `BlogIndexPage`. How can I restrict users from creating secondary index pages? E.g. is it possible to keep the `BlogIndexPage` from showing up as an option when creating a new page?

**Comment by timonweb** [comment]
Set `is_creatable = False` on your BlogIndex model class and you’re all set :)

**Comment by brylie** [comment]
Ok, so simple! This snippet would be a good aside in the intro tutorial, as I'm probably not the only person with this question ☺

**Comment by brylie** [comment]
Ah, really clever solution! This should be added to the docs, if it isn't there already (I am still reading). Could it be a 'Wagtail recipe'?

**Comment by timonweb** [comment]
I came up with it myself, don't know if it's in the docs.

**Comment by timonweb** [comment]
Here we go, I've created a quick tutorial on my blog: https://timonweb.com/tutorials/prevent-users-from-creating-certain-page-types-in-wagtail-cms/.

**Comment by sir-sigurd** [comment]
You could also try something like this

```
from django.utils.decorators import classproperty  # not sure if this is public API

class BlogIndexPage(Page):
    @classproperty
    def is_creatable(cls):
        return not cls.objects.exists()
```

**Comment by zerolab** [comment]
The 'unofficial way', as covered by tests is to override `can_create_at` and augment the parent check with a `not cls.objects.exists()`

https://github.com/wagtail/wagtail/blob/master/wagtail/tests/testapp/models.py#L862

Will work on adding `max_count` and updating the documentation to settle on one solution.

**Comment by gasman** [comment]
Implemented in #4852
